test/dhtest.c

```diff
@@ -79,11 +83,12 @@ static int dh_test(void)
     if (!TEST_true(BN_set_word(priv_key, 1234L))
         || !TEST_true(DH_set0_key(dh, NULL, priv_key)))
         goto err;
+    priv_key1 = priv_key, priv_key = NULL;
```

Comment:
Does this kind of expression adhere to the code style guidelines?

Reply:
I don't think we have a rule that forbids comma expressions ;-). But if anyone dislikes them, I will change it. I used it here to emphasize that the value moves from the one variable to the other.